docs-www/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@brainhubeu/gatsby-docs-kit-start",
+  "description": "Gatsby-Docs-Kit starter repository",
+  "version": "0.0.49",
+  "author": "Brainhub",
+  "license": "MIT",
+  "scripts": {
+    "build": "rm -rf .cache && rm -rf public && gatsby build --prefix-paths",
+    "develop": "rm -rf .cache && NODE_ENV=development gatsby develop",
+    "seed": "bash tools/seed.sh",
+    "serve": "export RC_ENV=__RC_ENV__; if [[ \"$RC_ENV\" == 'development' ]]; then yarn develop; else gatsby serve; fi",
+    "lint": "eslint --ext .jsx,.js .",
+    "lint:autofix": "eslint --ext .jsx,.js . --fix"
+  },
+  "dependencies": {
+    "@brainhubeu/gatsby-docs-kit": "3.0.8",
+    "@brainhubeu/react-file-input": "^0.7.21",
+    "gatsby": "2.20.10",
+    "gatsby-link": "2.3.1",
+    "lodash": "^4.17.15",
+    "react-fa": "^5.0.0"
```

Comment:
I cannot see `react-fa` usage here but it would be good to track these docs in GA.